### ISSUE TITLE
Configure Special Fields

### ISSUE THREAD

**Issue by kevinpostlewaite** [killbill/killbill-payu-latam-plugin#2]
Similar to https://github.com/killbill/killbill-cybersource-plugin/issues/3 (see for linked commit fixing this issue)

plugin_first_reference_id
plugin_second_reference_id
plugin_gateway_error

plugin_pm_id
plugin_pm_type
plugin_pm_cc_name
plugin_pm_cc_type
plugin_pm_cc_expiration_month
plugin_pm_cc_expiration_year
plugin_pm_cc_last_4
plugin_pm_address1
plugin_pm_address2
plugin_pm_city
plugin_pm_state
plugin_pm_zip
plugin_pm_country

**Comment by pierre** [comment]
Can you precise to which fields from the PayU response these should map?

**Comment by kevinpostlewaite** [comment]
No: some are obvious but I don't know for the others. I think Eric might know who to talk to to find out enough information to be able to map the reference IDs

**Comment by kevinpostlewaite** [comment]
payu_latam_transactions.txn_id has three values, separated by ;
First: authorization code returned by PSP
Second: transaction UUID assigned by PSP
Not sure what #3 is

**Comment by pierre** [comment]
> Not sure what #3 is

It's the [authorizationCode](https://github.com/pierre/active_merchant/blob/30320a6b1505bd09a45a0b3cf97295aded4407de/lib/active_merchant/billing/gateways/pay_u_latam.rb#L197).
